src/platform/forms/save-in-progress/ApplicationStatus.jsx

```diff
@@ -213,7 +213,7 @@ export class ApplicationStatus extends React.Component {
           >
             <p>Are you sure you want to start over?</p>
             <ProgressButton
-              onButtonClick={() => this.removeForm(formId)}
+              onButtonClick={() => this.removeForm(formId, formConfig.rootUrl)}
```

Comment:
Since it doesn't look like the `rootUrl` is used anywhere else in the formConfig, would it make more sense if we got it from `formConfig.saveInProgress.rootUrl`?

Reply:
Potentially, but I felt like `rootUrl` _could_ be used for more than SiP functionality, whereas everything in `saveInProgress` should _only_ be used for SiP functionality.